modules/store.js

```diff
@@ -17,11 +17,13 @@ export const reducer = handleActions({
   [BEGIN_GLOBAL_LOAD]: (state) => ({
     ...state,
     loaded: false,
+    loading: true,
```

Comment:
if there is a double load trigger - it means something has gone wrong, that should be investigated